printui/main.py

```diff
@@ -7,25 +7,24 @@
 import sys
 import logging
 import random
-import json
 import configparser
 import argparse
 import io
 import base64
 import functools
+import importlib.resources
 
 import PIL.ImageFont
 import PIL.ImageDraw
 import bottle
 import fontconfig
-import pkg_resources
 
 import brother_ql.conversion
 import brother_ql.raster
 
 from .printer import PrinterDevice, PrinterError
 
-TEMPLATE_DIR = [pkg_resources.resource_filename(__name__, 'views')]
+TEMPLATE_DIR = [importlib.resources.files(__package__).joinpath('views')]
```

Comment:
Note that `importlib.resources.files` does not return a directory path of type `pathlib.Path` if the package is loaded from a ZIP file. I suspect that Jinja2 and bottle.py will run into an exception in such cases. :/ The method [`pkg_resources.resource_filename`](https://setuptools.pypa.io/en/latest/pkg_resources.html#resource-extraction) seems to have extracted the ZIP on demand.